src/pages/resource/ReferencesPanel.tsx

```diff
@@ -20,31 +22,46 @@ const StyledFieldWrapper = styled.div`
 
 interface ReferencesPanelProps {
   goToNextTab: () => void;
+  saveResource: () => void;
   tabNumber: number;
 }
-export const ReferencesPanel: React.FC<ReferencesPanelProps> = ({ goToNextTab, tabNumber }) => {
+export const ReferencesPanel: React.FC<ReferencesPanelProps> = ({ goToNextTab, saveResource, tabNumber }) => {
   const errors = useSelector((store: RootStore) => store.errors);
   const { t } = useTranslation();
+  const [persistedFormData, setPersistedFormData, clearPersistedData] = useFormPersistor(
+    'resourceReferences',
+    emptyReferencesForm
+  );
 
   const initialFormikValues = {
-    publisher: { title: '', issn: '', level: '', publisher: '' },
-    referenceType: ReferenceType.PUBLICATION_IN_JOURNAL,
+    publisher: {
+      issn: (persistedFormData.publisher && persistedFormData.publisher.issn) || '',
+      level: (persistedFormData.publisher && persistedFormData.publisher.level) || '',
+      publisher: (persistedFormData.publisher && persistedFormData.publisher.publisher) || '',
+      title: (persistedFormData.publisher && persistedFormData.publisher.title) || '',
+    },
+    referenceType: persistedFormData.referenceType || ReferenceType.PUBLICATION_IN_JOURNAL,
   };
 
   return (
     <TabPanel
-      isHidden={tabNumber !== 2}
       ariaLabel="references"
-      goToNextTab={goToNextTab}
       errors={errors.referencesErrors}
-      heading={t('publication:heading.references')}>
+      goToNextTab={goToNextTab}
+      heading={t('publication:heading.references')}
+      isHidden={tabNumber !== 2}
+      onClickSave={() => {
+        saveResource();
+        clearPersistedData();
+      }}>
       <Box>
         <Formik
           enableReinitialize
           initialValues={initialFormikValues}
           onSubmit={(values, { setSubmitting }) => {
             setSubmitting(false);
-          }}>
+          }}
+          validate={(values: ReferencesFormData) => setPersistedFormData(values)}>
```

Comment:
ingen validering?

Reply:
ingenting som er påkrevd enda. blir sikkert noe etterhvert, men vet ikke hvordan denne taben skal se ut enda

Reply:
kan avt prøve `handleBlur` eller `onBlur`

Reply:
til å gjøre hva? setPersistedFormData blir kalt på `onBlur`

Reply:
skjønner. var bare litt missvisende at validate-funksjonen ikke validerer i stedet for å faktisk validere. Men hvis dette er veldig wip er det ikke så farlig

Reply:
ganske wip ja